stable/satisfactory/10.1.1/ix_values.yaml

```diff
@@ -0,0 +1,58 @@
+image:
+  repository: wolveix/satisfactory-server
+  pullPolicy: IfNotPresent
+  tag: v1.8
```

Comment:
You deliberately chose `v1.8` and not `v1.8.5` and removed the hash?
That way it should automatically take the latest version with the `v1.8` tag, right?
Does this also work if a new version with the tag `v1.8` is pushed later, but an older version with the same tag is already in use?

Reply:
I did it that was as a less aggressive version of just using latest. The maintainer has a pretty good automated management of tags so reversion won't be an issue.

However, it was mostly to reduce maintenance burden. If you implement the auto update scripts like that other guy uses, then version locking is fine and this change is less necessary.

Reply:
It will only pull the latest image when you reinstall the app (or manually delete the image from the system) because with pullPolicy: IfNotPresent, the system will detect that a v1.8 image is already present and won't download it again. It's smart to include the hash to prevent the app from breaking due to changes in the v1.8 series, even though v1.8 is a less aggressive tag than latest.

I recommend updating to the newest version (if it works for your setup) and including the hash to lock that version. While it's possible to set pullPolicy: Always to ensure you're always using the latest image, this can lead to rate limit issues on GitHub or other registries if a lot of people are pulling the image frequently. Some repositories may even block users who continuously pull without caching.

Reply:
i don't recommend doing hash locks unless you're going to actively maintain the repo. it's pretty trivial to go in and delete the existing image, but its a pain to manage a fork + pr to get a new version every time if it's just minor changes that don't require chart reworks.

Reply:
You make a **good point**, and I agree that rollbacks can be a quick fix. However, rollbacks generally only revert the *app*—not the content or save files. If an update changes the data format or structure, there’s a chance that the save files or even backups could be affected, especially if the app does automatic migrations or overwrites older backups.

The **chance of something breaking is low**, but personally, I wouldn’t take that risk. With my luck, I’d end up losing all the hours I’ve put into the game save. So I’d rather wait a bit longer or just use my own catalog while waiting, rather than skipping the version lock and hoping for the best.

But that’s just *my opinion*—everyone has different risk tolerance!